qtconsole/console_widget.py

```diff
@@ -1657,24 +1689,23 @@ def _event_filter_page_keypress(self, event):
         return False
 
     def _on_flush_pending_stream_timer(self):
-        """ Flush the pending stream output and change the
-        prompt position appropriately.
+        """ Flush pending text into the widget on console timer trigger.
         """
-        cursor = self._control.textCursor()
-        cursor.movePosition(QtGui.QTextCursor.End)
         self._flush_pending_stream()
-        cursor.movePosition(QtGui.QTextCursor.End)
 
     def _flush_pending_stream(self):
-        """ Flush out pending text into the widget. """
+        """ Flush pending text into the widget. Only applies to text that is pending
+            when the console is in the running state. Text printed when console is
+            not running is shown immediately, and does not wait to be flushed.
+        """
```

Comment:
```suggestion
        """
        Flush pending text into the widget.

        It only applies to text that is pending when the console is in the
        running state. Text printed when console is not running is shown
        immediately, and does not wait to be flushed.
        """
```

Improve formatting. I think this won't be done by auto-formatters.